--- libkcompactdisc-17.04.1/src/CMakeLists.txt.omv~	2017-05-10 03:53:31.433417404 +0200
+++ libkcompactdisc-17.04.1/src/CMakeLists.txt	2017-05-10 03:53:55.212664174 +0200
@@ -46,6 +46,8 @@ set(wmlib_SRCS
         wmlib/drv_toshiba.c
 )
 
+add_definitions(-std=gnu11)
+
 if (APPLE OR WIN32 OR CMAKE_SYSTEM_NAME STREQUAL GNU)
         set(USE_WMLIB false)
 else (APPLE OR WIN32 OR CMAKE_SYSTEM_NAME STREQUAL GNU)
